roles/loki/files/config.j2

```diff
@@ -1,16 +1,24 @@
 auth_enabled: false
 
 server:
-  http_listen_address: 127.0.0.1
+  http_listen_address: 172.16.34.1
```

Comment:
This could be a variable `loki_http_listen_address`, which could be set in `inventory` in `group_vars/all/vars.yml`, so it can be used also in Grafana config.

Similarly, we need to be 100% sure that there are no listeners for the GRPC (or other things) on any public IPs!